barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -52,8 +52,9 @@ acir_proofs::AcirComposer init(acir_format::acir_format& constraint_system)
 
 void init_reference_strings()
 {
-    // TODO(https://github.com/AztecProtocol/barretenberg/issues/811): Don't hardcode subgroup size
-    size_t subgroup_size = 32768;
+    // TODO(https://github.com/AztecProtocol/barretenberg/issues/811): Don't hardcode subgroup size. Currently set to
+    // max circuit size present in acir tests suite.
+    size_t subgroup_size = 262144;
```

Comment:
Search for the string in the code -> should be changed in bb.js as well

Reply:
done, thanks